apps/synapse-portal-framework/src/App.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { PropsWithChildren } from 'react'
+import { Outlet } from 'react-router-dom'
+import {
+  CookiesNotification,
+  SynapseToastContainer,
+} from 'synapse-react-client'
+import AppInitializer from './components/AppInitializer'
+import Footer from './components/Footer'
+import Navbar from './components/navbar/Navbar'
+
+export default function App(props: PropsWithChildren) {
```

Comment:
Extracted component from Portal.tsx. This `App` component renders the app layout & content. The `Portal` component renders all of the context and the new RouteProvider.

`Portal` does not _directly_ render `App`--`App` is included as a top level layout route.